common/traits/druid_forms.txt

```diff
@@ -100,8 +100,8 @@ master_talon_form = {
 	martial = 3
 	diplomacy = -5
 	command_modifier = {
-		speed = 0.3
-		damage = 0.3
+		speed = 0.2
+		damage = 0.25
```

Comment:
I'd also decrease this if you decrease pack form's damage buff, keeping the speed buff